### PR TITLE
Fix some typos [backport]

### PR DESCRIPTION
**Pull request by ghost**
There are also some other changes but I didn't include them because I think they're more opinionated - e.g. "datastructure" -> "data structure", "

### CODE REVIEW

File: doc/codeowners.rst [hunk 1]
```diff
@@ -13,31 +13,31 @@ library, araq is responsible for the compiler.
...
-======================         ======================================================
+===========================     ======================================================
+subsystem                       owner(s)
+===========================     ======================================================
+Parsing, Lexing                 araq
+Renderer                        cooldome, araq
+Order of passes                 cooldome
+Semantic Checking               araq
+Virtual machine                 jangko, GULPF, araq
+Sempass2: effects tracking      cooldome, araq
```
Comment (ghost) [inline doc/codeowners.rst:24]: This table was incorrectly rendered before because of this line

File: doc/docgen.rst [hunk 2]
```diff
@@ -244,7 +244,7 @@ Other Input Formats
 
 The *Nim compiler* also has support for RST (reStructuredText) files with
 the ``rst2html`` and ``rst2tex`` commands. Documents like this one are
-initially written in a dialect of RST which adds support for nim sourcecode
+initially written in a dialect of RST which adds support for nim source code
```
Comment (ghost) [inline doc/docgen.rst:247]: I checked - that's the only case where "sourcecode" is used, all other cases use "source code"

**Comment by ghost** [comment]
macos and ubuntu packages CI failure unrelated:
```
  FAIL: https://github.com/Vindaar/ggplotnim C
  Test "https://github.com/Vindaar/ggplotnim" in category "nimble-packages-1"
```

**Comment by Vindaar** [comment]
The heck, what's wrong with ggplotnim's CI again?

**Pull request merged**